lib/Runtime/Base/ScriptContext.cpp

```diff
@@ -1267,12 +1267,6 @@ if (!sourceList)
     void ScriptContext::SetIsClosed()
     {
         this->isClosed = true;
-#if ENABLE_NATIVE_CODEGEN
-        if (m_remoteScriptContextAddr)
-        {
-            JITManager::GetJITManager()->CloseScriptContext(m_remoteScriptContextAddr);
```

Comment:
If it is closed but not cleaned up, we don't need to reject the call. It should really only be for telling codegen that it can give up on any current JIT calls (we have periodic checks for IsClosed during codegen).

---
In reply to: [83088845](https://github.com/Microsoft/ChakraCore/pull/1728#discussion_r83088845) [](ancestors = 83088845,83082897)